jsx/FilterableDataTable.js

```diff
@@ -57,13 +58,17 @@ class FilterableDataTable extends Component {
       />
     );
 
-    const dataTable = (
+    const dataTable = !isNaN(this.props.loading) && this.props.loading < 100 ? (
+      <LoadingBar progress={this.props.loading}/>
+    ) : (
       <DataTable
         data={this.props.data}
         fields={this.props.fields}
         filter={this.state.filter}
         actions={this.props.actions}
+        forms={this.props.forms}
```

Comment:
not in **24.1**

_created to more easily seamlessly render forms. Can probably find a work-around when pushing to main, although could be potentially nice to include_

include as **CBIG override** and potentially push to **main**